lib/target.ml

```diff
@@ -42,20 +322,19 @@ module type Target = sig
 

```

Comment:
Can we break implementations of Target out into their own files?

Reply:
core and v1model now have their own files, with any desired architecture to follow. I also started thinking through the details of making a "corize" functor if that's something we still want to explore.